ckanext/depositar_theme/helpers.py

```diff
@@ -67,3 +72,43 @@ def send_reg_link(user):
     subject = subject.split('\n')[0]
 
     mail_user(user, subject, body)
+
+@core_helper
+def get_format_count():
+    results = logic.get_action('package_search')({}, {})['results']
+    format_list = []
+    for res in results:
+        resourceList = res['resources']
+        for resource in resourceList:
+            format_list.append(resource['format'])
+
+    format_count = dict((fmt, format_list.count(fmt)) for fmt in format_list)
+    return list(format_count.items())
+
+@core_helper
+def get_download_count():
+    package_list = logic.get_action('package_list')({},{})
+    context = {
+        "model": model,
+        "session": model.Session,
+    }
+    download_cnt = 0
+    for pkg in package_list:
+        data = {'id': pkg, 'include_tracking': True}
+        resources = logic.get_action('package_show')(context ,data)['resources']
```

Comment:
除非要驗證使用者身分等情況，毋需傳入 context

```suggestion
        resources = tk.get_action('package_show')({}, data)['resources']
```